src/config/config.ts

```diff
@@ -19,10 +19,10 @@ interface EtherscanNetworkApiDetails {
   hostname: string;
 }
 
-const ropstenGsnRelayConfig = {
-  relayHub: "0x29e41C2b329fF4921d8AC654CEc909a0B575df20",
-  stakeManager: "0x762A4D5F51d8b2F9bA1B0412B45687cE0EfFD92B",
-  forwarder: "0x25CEd1955423BA34332Ec1B60154967750a0297D",
+const goerliGsnRelayConfig = {
+  relayHub: "0x1F3d1C33977957EA41bEdFDcBf7fF64Fd3A3985e",
+  stakeManager: "0xd494924348e91433218506bc0ed02b3a2d83ef0e",
+  forwarder: "0xd9c1a99e9263B98F3f633a9f1A201FA0AFC2A1c2",
   gasPrice: 20000000000, // 20 Gwei
 };
```

Comment:
just curious, where did u get these values from

Reply:
from. here: https://docs-v2.opengsn.org/contracts/addresses201.html#goerli-testnet

Reply:
mm not familiar with gsn portion. will approve first, but if @superical has any comments about supporting gsn in future can chip in

Reply:
Are we really utilising GSN in creator? I remember asking something similar in https://github.com/TradeTrust/document-creator-website/pull/262#discussion_r994157739 when I first saw that GSN bit. @isaackps has clarified that it was previously added but not used. Should we remove it if we aren’t using it anymore? But of course if there are still plans to use it then it makes sense to keep it.

Reply:
@superical ya for this we need to discuss with PO. So if wanna remove it I think we do it in another branch ba.